bedrock/firefox/templates/firefox/hello/index.html

```diff
@@ -52,9 +52,6 @@
 data-try-now="{{ _('Try it now') }}"
 {% endblock %}
 
-{% block tabzilla_css %}{% endblock %}
-{% block tabzilla_js %}{% endblock %}
-
```

Comment:
I'm okay leaving this page using `site_css`, as long as we make a note that it needs to be updated when we update tabzilla in the Fx family nav.

Do we have a bug for that yet? If not, can you file one and note this page will need its CSS block/bundle updated?